templates/shortcode-receipt.php

```diff
@@ -21,6 +21,8 @@
 $cart    = edd_get_payment_meta_cart_details( $order->id, true );
 $user    = edd_get_payment_meta_user_info( $order->id );
 $status  = edd_get_payment_status( $order, true );
+
+do_action( 'edd_payment_receipt_before_table', $payment, $edd_receipt_args );
```

Comment:
I like the idea of this hook to promote more freeform (and valid) HTML. 👍 

This should be documented to ensure it is used, and used properly: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
I'm all for documentation, but in this case since none of the other actions in this template have documentation it would seem strange to me to require it for this action. That may be better handled as a project/pull request in itself. I was just seeing plugins (including your own) abusing another hook for getting the desired result, producing invalid HTML and this is a proper alternative.

Reply:
Existing hooks and filters will eventually need inline documentation (likely as a separate project like you mentioned) so it makes sense to me to go ahead and document any newly introduced hooks and filters added before that project can be started.

I think it is especially important to ensure new hooks and filters in overridable template files have documentation to help theme and plugin authors from using the wrong one or needlessly overriding the entire file.